### ISSUE TITLE
Journal & Discussions - User Can't Delete his own post?

### ISSUE THREAD

**Issue by YunusDange** [arshadansari27/fitranginew#181]
i have tried deleting some Blogs and discussions as a user ho created that...but it shows that successfully deleted but it is still there in the main section.....only after deleting it through admin it goes

user should allowed to delete his/her own blogs / posts and discussions

**Comment by YunusDange** [comment]
It's working for Journal

but i think...user doesn't have a option to delete discussions....have to create that

**Comment by arshadansari27** [comment]
Where will we give the user ability to delete his discussions? in all the row views? Like we have for post? a small 'X' on top right?

**Comment by YunusDange** [comment]
Yup
i think...that will work...
or else to make it more useful....we can even add new TAB in discussions section...Like we have in journal  "My Blogs" 
same TAB in Discussions "My Discussions"

**Comment by arshadansari27** [comment]
Let's just allow delete from the editor view..

My Discussions tab should be a separate ticket with a different milestone...

**Comment by YunusDange** [comment]
Ok
but delete option from editor view will not work....once user post a discussion ...then how he will go to editor view again..after few days..there is no option for this yet..

so we can have delete option in all the row views... Like we have for post....a small 'X' on top right..on all the discussions written by that particular user..

.but keep one more confirmation popup in between...to get a yes/no confirmation before deleting (as in post view there is no confirmation in between...so if user click on that X button accidentally then also it will get deleted ..)

so we should avoid that here
